collector/kernel/buffered_poller.cc

```diff
@@ -79,7 +80,7 @@ BufferedPoller::BufferedPoller(
       nat_handler_(writer_, log),
       nic_poller_(nic_poller),
       tcp_socket_table_ever_full_(false),
-      tslot_(1e9, 16),
+      tslot_(socket_stats_interval_sec * 1000 * 1000 * 1000, 16),
```

Comment:
Could this be `socket_stats_interval_sec * 1e9`, making `socket_stats_interval_sec` an integer?

Reply:
It certainly could be.  I didn't want to force the minimum interval to be 1 second in case we ever for some reason want to make it less than that so I went with double for that reason, along with the fact that the fast_div tslot_ takes a double.  I could change this to take an int # of milliseconds - and call it socket_stats_interval_ms - or if we know for sure that we never want this to go below 1 sec. I could change it to an int # of seconds.  Let me know your thoughts.

Reply:
Since the underlying time is in nanoseconds, and with the filter-ns in integer nanoseconds, the most straightforward might be to accept the timeslot length in integer nanoseconds.

Reply:
Would also alleviate the need for multiplication by a constant mentioned above..